.travis.gofmt.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd "$(dirname $0)"
+if [ -n "$(go fmt ./...)" ]; then
```

Comment:
I yanked [google/gopacket](https://github.com/google/gopacket/blob/f9d6df4cf07e221be4221834a9f903ee7688ba7a/.travis.gofmt.sh)'s approach.

To clarify how this works: `if [ -n "$(go fmt ./...)" ]` writes the fixes and outputs the files (making this conditional true thus failing Travis). So the committer likely doesn't need to manually run gofmt again - Travis failure output provides an explanation and the comitter can simply verify/commit the formatting fixes.

Reply:
Lovely!